### PR TITLE
ALCB Token Specification

### PR DESCRIPTION
**Pull request by chgorman**
Draft of ALCB Token Spec.

### CODE REVIEW

File: specifications/alcb-token.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+# ALCB Token Specification
+
+The ALCB Token is the utility token of AliceNet.
+It follows a bonding curve which is parameterized by
+the total ETH reserves and specifies the ALCB/ETH (inverse price);
+this is different from the standard parameterization,
+which tracks the total token supply and specifies
+the ETH/Token (price).
+
+## Minting and Burning
```
Comment (diligentis) [inline specifications/alcb-token.md:17]: Probably be a good idea to have the example chart in here too to help follow along with the math.

**Review by diligentis — CHANGES_REQUESTED** [review]
One nit otherwise looks good!

File: specifications/alcb-token.md (hunk 1 above)
Comment (chgorman) [reply specifications/alcb-token.md:17]: Examples for showing how many tokens are received when purchasing at different ETH reserves?

Comment (chgorman) [reply specifications/alcb-token.md:17]: Added some specific numbers. Need to discuss more.

**Pull request merged**